### PR TITLE
Fix issues reported by the 'unused', 'staticcheck', 'gosimple' linters. closes #1360

### LINKED ISSUE #1360: Fix issues reported by the 'unused' and 'gosimple' linters

### PR DESCRIPTION
**Pull request by pskrbasu**
@kaidaguerre Is this function needed for reporting?
```
cmd/report.go:14:6: func `runReportCmd` is unused (unused)
func runReportCmd(cmd *cobra.Command, args []string) {
     ^
```

### CODE REVIEW

**Comment by kaidaguerre** [comment]
yup - that function is used in the (soon to be merged)reporting branch

File: query/metaquery/completers.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func Complete(input *CompleterInput) []prompt.Suggest {
 
 func completerFromArgsOf(cmd string) completer {
 	return func(input *CompleterInput) []prompt.Suggest {
-		metaQueryDefinition, _ := metaQueryDefinitions[cmd]
+		metaQueryDefinition := metaQueryDefinitions[cmd]
```
Comment (e-gineer) [inline query/metaquery/completers.go:34]: Is this a change that continues to ignore an error? If so, shouldn't we handle the error?

Comment (pskrbasu) [reply query/metaquery/completers.go:34]: `metaQueryDefinitions` is a map[string]metaQueryDefinition. We are just assigning the value here.

Comment (kaidaguerre) [reply query/metaquery/completers.go:34]: to expand: in this map syntax ```metaQueryDefinition, _ := metaQueryDefinitions[cmd]``` the second return val is a boolean indicating whether the value was found

File: steampipeconfig/parse/decode.go [hunk 2]
```diff
@@ -287,7 +288,8 @@ func decodeParam(block *hcl.Block, runCtx *RunContext, parentName string) (*modc
 		diags = append(diags, valDiags...)
 	}
 	if attr, exists := content.Attributes["default"]; exists {
-		v, diags := attr.Expr.Value(runCtx.EvalCtx)
+		var v cty.Value
```
Comment (kaidaguerre) [inline steampipeconfig/parse/decode.go:291]: what is this change for?

**Pull request merged**